src/shared-components/dropdown/index.tsx

```diff
@@ -5,6 +5,20 @@ import MobileDropdown from './mobileDropdown';
 import DesktopDropdown from './desktopDropdown';
 import allowNullPropType from '../../utils/allowNullPropType';
 
+export type OptionType = {
+  value: string;
+  label: string;
+  disabled?: boolean;
+};
+
+type DropdownProps = {
+  value?: string;
+  onChange: (option: OptionType) => void;
+  options: OptionType[];
+  optionsContainerMaxHeight?: string;
+  textAlign: 'left' | 'center';
```

Comment:
Based on defaultProps it looks like `textAlign` should be potentially optional.

Also, `onChange` probably should've never been prop-typed as optional. All our usage requires it and this component makes no sense without it, so we can remove the `defaultProps` for it and make the PropTypes `isRequired`.